superset/viz.py

```diff
@@ -612,7 +611,7 @@ def query_obj(self):
         return None
 
     def get_df(self, query_obj=None):
-        return None
+        return pd.DataFrame()
```

Comment:
I added this for consistency, i.e. `get_df(...)` should return a `pd.DataFrame()`.